package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -0,0 +1,337 @@
+"""
+Secondary structure assignment (helix, sheet and loop) --- :mod:`MDAnalysis.analysis.dssp`
+=========================================================================================
+
+:Author: Egor Marin
+:Year: 2024
+:Copyright: LGPL v2.1+
+
+.. versionadded:: 2.8.0
+
+The module contains code to build hydrogend bond contact map,
+and use it to assign protein secondary structure (:class:`DSSP`).
+
+This module uses the python version of the original algorithm by Kabsch & Sander (1983),
+re-implemented by @ShintaroMinami and available under MIT license
+[here](https://github.com/ShintaroMinami/PyDSSP/tree/master#differences-from-the-original-dssp)
+Note that this implementation does not discriminate different types of beta-sheets,
+as well as different types of helices, meaning you will get 3_10 helices
+and pi-helices labelled as "helix" too.
+
+
+Using original `pydssp`
+----------------------
+
+The default implementation uses the original `pydssp` (v.0.9.0) code,
+rewritten without usage of `einops` library and hence having no dependencies.
+If you want to explicitly use `pydssp` (or its particular version), install it
+to your current environment with `python3 -m pip install pydssp`. Please note
+that using `batch` parameter, as well as `torch.Tensor`, would still be
+inaccessible.
```

Comment:
A bit confusing, I'd rewrite
```suggestion
that the way MDAnalysis uses pydssp does not support pydssp's capability
for batch processing or its use of the pytorch library.
```